### ISSUE TITLE
Resetting the view sometimes doesn't work

### ISSUE THREAD

**Issue by fkue587** [nebelorz/NW-Auto-Fishing#1]
Resetting the view sometimes doesn't work
Prevention of offline movement does not work

**Comment by nebelorz** [comment]
I'll check these issues as soon as I can, though I didn't used this script since past October, so maybe some things have changed on New World and that's the reason it's not working properly

**Comment by fkue587** [comment]
Haha I thought you just updated the app in 22 days! Thanks for sharing it would be great if you could update again!

**Comment by nebelorz** [comment]
> Haha I thought you just updated the app in 22 days! Thanks for sharing it would be great if you could update again!

Yup! Just when I left the game and stopped using this program 😋

Anyways, I'm testing the view resetting that you said, and the only thing I can do to avoid the "AFK disconnect" is to lower the number of casts to repair.

**Comment by nebelorz** [comment]
> Haha I thought you just updated the app in 22 days! Thanks for sharing it would be great if you could update again!

Okay, I just updated the program to avoid all the issues you posted🙃 now it should avoid the AFK disconnection and should reset the camera correctly everytime. 
While checking those issues, I just optimized the code, and it should generally run faster now.

I just tested it around, but if you're going to use it in depth, feel free to tell me any other issues you find!
